pygsti/circuits/circuit.py

```diff
@@ -185,7 +199,7 @@ def to_label(x):
     """
     if isinstance(x, _Label): return x
     # # do this manually when desired, as it "boxes" a circuit being inserted
-    #elif isinstance(x,Circuit): return x.to_circuit_label()
+    elif isinstance(x,Circuit): return x.to_label()
```

Comment:
Neither Nick nor I remember making this change. The commented-out branch was broken. The uncommenting seems valid based on the to_label member function of Circuit objects.